packages/mjml-text/README.md

```diff
@@ -1,6 +1,11 @@
 ## mjml-text
 
-This tag allows you to display text in your email.
+Most important, this element places text in your email.
+
+Also important, this element supports placement of raw HTML tags
+  and their attributes anywhere within.
+Don't forget to encode special characters to avoid unexpected
```

Comment:
This was an issue in MJML3, and it's no longer the case so we can remove this phrase

Reply:
"Don't forget to encode special characters ...": gone